src/components/PostPageDetail/index.js

```diff
@@ -568,7 +569,7 @@ const PostPageDetailIdComponent = (props) => {
   };
 
   const handleUpdateVote = () => {
-    getComment();
+    getComment(false, true);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `getComment` function is now being called with two arguments instead of none. Make sure that the `getComment` function can handle these parameters correctly and that they are used appropriately within the function.
```diff
-    getComment();
+    getComment(false, true);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->